### ISSUE TITLE
Automated Target Priority Bug

### ISSUE THREAD

**Issue by Blackjack202** [Sandrem/FlyCasual#2996]
Hi,

A ship I had with Automated Target Priority is at range 0 of one enemy and range 1 of another. According to the game, I'm unable to attack either. I can't attack the range 1 enemy because ATP says I have a closer enemy. I can't attack the range 0 enemy since, well, it's at range 0.

This can't be right, can it? ATP only makes you attack the nearest valid target, and range 0 targets shouldn't be valid, as far as I can tell.

PS: Thanks a billion for this incredible game- it's brought me and my friends hours of joy, especially during shutdown.

![image](https://user-images.githubusercontent.com/62766247/98749414-7a4b5a00-2370-11eb-8861-45edc4f67105.png)

**Comment by Blackjack202** [comment]
Believe it or not, I just stumbled across another issue with ATP from the same game:

Before attacking, I took a calculate token from ATP. I missed that attack, which placed another token on ATP. Then, my ship remained selected, as though I needed to choose a target, and the game halted, since I could not select another target to attack again and there was no "skip" option visible. 

![image](https://user-images.githubusercontent.com/62766247/98749997-c054ed80-2371-11eb-9d6b-0e21fa5c5fa3.png)
